payments/mobile-pay/other-features.md

```diff
@@ -16,7 +16,699 @@ sidebar:
+Host: {{ page.apiUrl }}
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+`
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/mobilepay/payments/{{ page.paymentId }}",
+    "authorizations": {
+        "id": "/psp/mobilepay/payments/{{ page.paymentId }}/authorizations",
+        "authorizationList": [{
+            "id": "/psp/mobilepay/payments/{{ page.paymentId }}/authorizations/{{ page.transactionId }}",
+            "paymentToken": "{{ page.paymentToken }}",
+            "maskedPan": "123456xxxxxx1234",
+            "expireDate": "mm/yyyy",
+            "panToken": "{{ page.transactionId }}",
+            "cardBrand": "Visa",
+            "cardType": "Credit Card",
+            "issuingBank": "UTL MAESTRO",
+            "countryCode": "999",
+            "acquirerTransactionType": "3DSECURE",
+            "acquirerStan": "39736",
+            "acquirerTerminalId": "39",
+            "acquirerTransactionTime": "2017-08-29T13:42:18Z",
+            "authenticationStatus": "Y",
+            "transaction": {
+                "id": "/psp/mobilepay/payments/{{ page.paymentId }}/transactions/{{ page.transactionId }}",
+                "created": "2018-09-14T01:01:01.01Z",
+                "updated": "2018-09-14T01:01:01.03Z",
+                "type": "Authorization",
+                "state": "Initialized",
+                "number": 1234567890,
+                "amount": 1000,
+                "vatAmount": 250,
+                "description": "Test transaction",
+                "payeeReference": "AH123456",
+                "failedReason": "",
+                "failedActivityName": "Authorize",
+                "failedErrorCode": "REJECTED_BY_ACQUIRER",
+                "failedErrorDescription": "General decline, response-code: 05",
+                "isOperational": true,
+                "operations": []
+            }
+        }]
+    }
+}
+```
+
+{:.table .table-striped}
+| Property                      | Type     | Description                                                                           |
+| :---------------------------- | :------- | :------------------------------------------------------------------------------------ |
+| `payment`                     | `string` | The relative URI of the payment this authorization transactions resource belongs to.  |
+| `authorizations`              | `object` | The authorization object.                                                             |
+| └➔&nbsp;`id`                  | `string` | The relative URI of the current authorization transactions resource.                  |
+| └➔&nbsp;`authorizationList`   | `array`  | The array of authorization transaction objects.                                       |
+| └➔&nbsp;`authorizationList[]` | `object` | The authorization transaction object described in the `authorization` resource below. |
```

Comment:
Gief transaction include from #114 pl0x?

Reply:
It's included and used